src/app/lib/config/services/arabic.js

```diff
@@ -64,17 +64,17 @@ export const service = {
     showAdPlaceholder: true,
     showRelatedTopics: true,
     podcastPromo: {
-      title: 'بودكاست',
-      brandTitle: 'بي بي سي إكسترا',
+      title: 'البودكاست',
+      brandTitle: 'بودكاست "عَلاقات"',
       brandDescription:
-        'بي بي سي إكسترا بودكاست يناقش كل أمور حياتنا اليومية، يأتيكم ثلاثة أيام أسبوعياً',
+        'بودكاست أسبوعي يناقش العلاقات العاطفية باختلافها ويسلط الضوء على تحدياتها وحلولها المتاحة',
       image: {
-        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p09t9xpv.jpg',
-        alt: 'بي بي سي إكسترا',
+        src: 'https://ichef.bbci.co.uk/images/ic/3000x3000/p09m72gz.jpg',
```

Comment:
Sorry just noticed this - we shouldn't use 3000x3000px images for this

If you replace this with `$recipe` as per the old configs, the promo will make these images and appropriate size

https://github.com/bbc/simorgh/blob/fe2e64aa4dfb01f69b6b3d4e1ad3456754ca4971/src/app/containers/PodcastPromo/index.jsx#L20

```suggestion
        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p09m72gz.jpg',
```

Reply:
👍

Reply:
Thank you for spotting this :D. I just made the changes, will talk about this to the rest of the team who was working on this PR on Monday